### PR TITLE
[13.0] [MIG] contract_sale: Migration to v13.0 

### PR DESCRIPTION
**Pull request by victoralmau**
Required https://github.com/OCA/contract/pull/533

@Tecnativa TT26074

### CODE REVIEW

File: contract_sale/migrations/12.0.2.0.0/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        (
+            "contract_sale.account_analytic_contract_line_manager",
+            "contract_sale.contract_template_line_manager",
+        ),
+        (
+            "contract_sale.account_analytic_account_contract_salesman",
+            "contract_sale.contract_contract_salesman",
+        ),
+    ]
+    openupgrade.rename_xmlids(cr, xmlids_to_rename)
```
Comment (joao-p-marques) [inline contract_sale/migrations/12.0.2.0.0/pre-migration.py]: Shouldn't you remove this migration script?

File: contract_sale/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1,7 @@
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+  * Luis M. Ontalba
+  * Pedro M. Baeza
```
Comment (joao-p-marques) [inline contract_sale/readme/CONTRIBUTORS.rst:4]: You forgot to add your name to `CONTRIBUTORS`

**Review by joao-p-marques — COMMENTED** [review]
Just some small comments

**Comment by joao-p-marques** [comment]
Also, check Travis for errors

**Comment by victoralmau** [comment]
> Also, check Travis for errors

Travis always error now because `contract` addon is required (https://github.com/OCA/contract/pull/533)

**Review by joao-p-marques — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please rebase

**Review by sbejaoui — APPROVED** [review]

File: contract_sale/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Contract from Sale",
+    "version": "13.0.1.0.0",
+    "category": "Sales",
+    "author": "Tecnativa, " "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/contract",
+    "depends": ["sale", "contract"],
+    "development_status": "Stable",
```
Comment (pedrobaeza) [inline contract_sale/__manifest__.py]:
```suggestion
    "development_status": "Production/Stable",
```

File: contract_sale/views/abstract_contract_line.xml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+            name="name"
+        >contract.abstract.contract.line form view (in contract_sale)</field>
+        <field name="model">contract.abstract.contract.line</field>
+        <field
+            name="inherit_id"
+            ref="contract.contract_abstract_contract_line_form_view"
+        />
+        <field name="arch" type="xml">
+            <field name="discount" position="attributes">
+                <attribute name="groups">sale.group_discount_per_so_line</attribute>
```
Comment (pedrobaeza) [inline contract_sale/views/abstract_contract_line.xml]: This group has changed to module `product` (which should be that way in first place):

```suggestion
                <attribute name="groups">product.group_discount_per_so_line</attribute>
```

File: contract_sale/views/contract.xml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+        <field name="model">contract.contract</field>
+        <field name="inherit_id" ref="contract.contract_contract_form_view" />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//field[@name='contract_line_ids']/tree/field[@name='discount']"
+                position="attributes"
+            >
+                <attribute
+                    name="groups"
+                    add="sale.group_discount_per_so_line"
```
Comment (pedrobaeza) [inline contract_sale/views/contract.xml]:
```suggestion
                    add="product.group_discount_per_so_line"
```

File: contract_sale/views/contract_line.xml [hunk 6]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!--TREE view-->
+    <record id="contract_line_tree_view" model="ir.ui.view">
+        <field name="name">contract.template.line tree view (in contract_sale)</field>
+        <field name="model">contract.line</field>
+        <field name="inherit_id" ref="contract.contract_line_tree_view" />
+        <field name="arch" type="xml">
+            <field name="discount" position="attributes">
+                <attribute name="groups">sale.group_discount_per_so_line</attribute>
```
Comment (pedrobaeza) [inline contract_sale/views/contract_line.xml]:
```suggestion
                <attribute name="groups">product.group_discount_per_so_line</attribute>
```

File: contract_sale/views/contract_template.xml [hunk 7]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!--FORM view-->
+    <record id="contract_template_form_view" model="ir.ui.view">
+        <field name="name">contract.template form view (in contract_sale)</field>
+        <field name="model">contract.template</field>
+        <field name="inherit_id" ref="contract.contract_template_form_view" />
+        <field name="arch" type="xml">
+            <field name="discount" position="attributes">
+                <attribute name="groups">sale.group_discount_per_so_line</attribute>
```
Comment (pedrobaeza) [inline contract_sale/views/contract_template.xml]:
```suggestion
                <attribute name="groups">product.group_discount_per_so_line</attribute>
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
I'm now thinking that the discount group should be in base `contract` module, as it's a group available at that level. Can you please remove it from here and add it to contract in this PR?

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**